tests/browser/browser-compatibility.spec.ts

```
@@ -0,0 +1,123 @@
+    // Check that the page title is not empty
+    const title = await page.title()
+    console.log(`Page title: ${title}`)
+    expect(title).not.toBe('')
+
+    // Check for main content
+    await expect(page.locator('main')).toBeVisible()
+
+    // Create directory for screenshots
+    await page.evaluate(() => {
+      try {
+        require('fs').mkdirSync('./test-results/browser-compat', {
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)